workspaces/3scale/plugins/3scale-backend/package.json

```diff
@@ -33,22 +33,27 @@
     "postversion": "yarn run export-dynamic",
     "prepack": "backstage-cli package prepack",
     "start": "backstage-cli package start",
-    "test": "backstage-cli package test --passWithNoTests --coverage",
-    "tsc": "tsc"
```

Comment:
`tsc` no longer required?

Reply:
This command fails with: 

```
❯ yarn tsc
command not found: tsc
```

And I took a look another backend plugins - they don't have this command. I guess typical dev flow in this repository - using `yarn build` instead of `yarn tsc`. That's why I removed it.